src/extension.ts

```diff
@@ -160,6 +162,8 @@ export function activate(context: vscode.ExtensionContext): void {
             void vscode.env.openExternal(vscode.Uri.parse('https://github.com/microsoft/vscode-edge-devtools/blob/master/README.md'));
         }));
     void vscode.commands.executeCommand('setContext', 'titleCommandsRegistered', true);
+    reportExtensionSettings(telemetryReporter);
+    vscode.workspace.onDidChangeConfiguration(event => reportChangedExtensionSetting(event, telemetryReporter));
```

Comment:
The `vscode.ConfigurationChangeEvent` only contains one function which is `affectsConfiguration(setting)` which returns a bool that checks whether the changed event matches the passed in setting string.